core/src/test/kotlin/org/stellar/anchor/sep31/RefundPaymentTest.kt

```diff
@@ -34,7 +34,32 @@ class RefundPaymentTest {
   }
 
   @Test
-  fun test_toSep31RefundPayment() {
+  fun test_toDatabaseEntity() {
+    // mock the PlatformApi RefundPayment object
+    val mockPlatformApiRefundPayment =
+      org.stellar.anchor.api.shared.RefundPayment.builder()
+        .id("A")
+        .idType(org.stellar.anchor.api.shared.RefundPayment.IdType.STELLAR)
+        .amount(Amount("50", stellarUSDC))
+        .fee(Amount("4", stellarUSDC))
+        .refundedAt(Instant.now())
+        .refundedAt(Instant.now())
+        .build()
+
+    // mock the SEP-31 RefundPayment object we want
+    val wantRefundPayment = PojoSep31RefundPayment()
+    wantRefundPayment.id = "A"
+    wantRefundPayment.amount = "50"
+    wantRefundPayment.fee = "4"
+
+    val gotSep31RefundPayment =
+      RefundPayment.callbackConverter(sep31TransactionStore, stellarUSDC)
```

Comment:
This would normally be called `converter()`. However, the `RefundPayment` class need to have two converters. One for CallbackAPI, another for SEP response.